src/components/SlateEditor/plugins/campaignBlock/CampaignBlockForm.tsx

```diff
@@ -63,9 +63,11 @@ const rules: RulesType<CampaignBlockFormValues> = {
   link: {
     required: true,
     url: true,
+    onlyValidateIf: (value) => !!value.linkText,
```

Comment:
Her tror jeg du må oppdatere til value.link 👍

Reply:
har satt det som linkText for at den skal kreve urlen hvis teksten er der. dvs det funker sånn:
- har du bare tekst, så krever den url med gylding lenke
- har du bare url, så krever den tekst
- er begge tomme så kan du lagre
- hvis begge er skrevet inn og url er validert så kan du lagre

Reply:
Vi burde jo vise feilmelding her, litt rar oppførsel at feilmelding for lenke-feltet først dukker opp når man fyller ut lenketekst-feltet
<img width="1078" alt="Skjermbilde 2023-12-21 kl  15 56 59" src="https://github.com/NDLANO/editorial-frontend/assets/26788204/5ead4a18-565c-4ddd-b38f-ebb0b4e10f64">
<img width="1092" alt="Skjermbilde 2023-12-21 kl  15 59 09" src="https://github.com/NDLANO/editorial-frontend/assets/26788204/e5cce474-5bc6-4bac-8ead-9484f87b378d">

Reply:
Altså oppdatere til:  ` onlyValidateIf: (value) => !!value.linkText || !!value.link,` for link
<img width="1082" alt="Skjermbilde 2023-12-21 kl  15 59 59" src="https://github.com/NDLANO/editorial-frontend/assets/26788204/81f43d43-ae20-4f24-ad3b-41d8e084cfcf">

Reply:
enig 👍